cohorts/cohort.py

```diff
@@ -638,16 +755,27 @@ def load_effects(self, patients=None, only_nonsynonymous=False,
              Dictionary of patient_id to varcode.EffectCollection
         """
         filter_fn = first_not_none_param([filter_fn, self.filter_fn], no_filter)
+        if filter_fn is None:
```

Comment:
In fact, this code is repeated enough that I think it should be extracted into a `get_filter_fn_name(default_name="None")` function?